### ISSUE TITLE
Pass S3 credentials via environment variables

### ISSUE THREAD

**Issue by Totktonada** [knazarov/mkrepo#20]
Now we can use `--s3-*` options, but this way the secrets appear in the cmdline and so in the `ps aux` output.

It is usual and convenient to pass secrets via environment.

**Comment by NickVolynkin** [comment]
Good point, env vars are more secure and more convenient in GH workflows. (Although, a little less explicit).
